package.json

```diff
@@ -12,6 +12,9 @@
   "bugs": {
     "url": "https://github.com/env0/winston-aws-dynamodb/issues"
   },
+  "scripts": {
+    "publish": "yarn publish"
```

Comment:
Any reason for `yarn` here instead of `npm`?

Reply:
because all our command with yarn. But I don't think is important.